### PR TITLE
maintain(recipe detail view): fixed detail view issues

### PR DESCRIPTION
**Pull request by Jan0H0Punkt**

### CODE REVIEW

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html
Comment (crookoo) [inline frontend/src/app/recipe/components/detail/recipe-detail.component.html:1]: I had a look into the issue of the too big headlines and found that Material is overriding the typography settings of Bootstrap. Bootstrap has a well tuned font size setting for headlines. So I suggest we get rid of the material typography settings for the moment and see how it looks with pure Bootstrap in this place. To achieve that just get rid of the CSS class mat-typography in the body tag of the index.html:
`<body class="mat-typography">` needs to be changed to `<body>`

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html [hunk 1]
```diff
@@ -1,35 +1,32 @@
 @if (_recipe$ | async; as recipe) {
-<h1>{{recipe.name}}</h1>
+  <h4>{{ recipe.name }}</h4>
```
Comment (crookoo) [inline frontend/src/app/recipe/components/detail/recipe-detail.component.html:2]: When we have pure Bootstrap CSS for our typography we should leave h1 here.

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html [hunk 2]
```diff
@@ -1,35 +1,32 @@
...
-            }
-        </ul>
-    </div>
+  <div class="row">
     <div class="col-md-4">
-        <h3>Ingredients:</h3>
-        <ul>
-            @for (ingredient of recipe.ingredients; track $index) {
-            <li>{{ingredient}}</li>
+      <h6>Zutaten:</h6>
```
Comment (crookoo) [inline frontend/src/app/recipe/components/detail/recipe-detail.component.html:12]: Lets put h2 here.

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html [hunk 3]
```diff
@@ -1,35 +1,32 @@
...
+      <ul>
+        @for (ingredient of recipe.ingredients; track $index) {
+          <li>{{ ingredient }}</li>
+        }
+      </ul>
+    </div>
+    <div class="col-md-8">
+      <h6>Beschreibung:</h6>
```
Comment (crookoo) [inline frontend/src/app/recipe/components/detail/recipe-detail.component.html:20]: I would prefer "Anleitung" or better "Zubereitung"

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html [hunk 4]
```diff
@@ -1,35 +1,32 @@
...
-            @for (block of recipe.instructions; track $index) {
-            <p>
-              @for(line of block; track $index){
-                <li>{{ line }}</li>
-              }
-            </p>
-            }
-        </ul>
-    </div>
+  <div class="row">
```
Comment (crookoo) [inline frontend/src/app/recipe/components/detail/recipe-detail.component.html:10]: Why another row here?

**Review by crookoo — CHANGES_REQUESTED** [review]

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html (hunk 2 above)
Comment (Jan0H0Punkt) [reply frontend/src/app/recipe/components/detail/recipe-detail.component.html:12]: I think h2 is to big.

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html (hunk 4 above)
Comment (Jan0H0Punkt) [reply frontend/src/app/recipe/components/detail/recipe-detail.component.html:10]: It's so on big screens the two list get rendered side by side instead below each other.

File: frontend/src/app/recipe/components/detail/recipe-detail.component.html (hunk 2 above)
Comment (crookoo) [reply frontend/src/app/recipe/components/detail/recipe-detail.component.html]: h2 is needed on the accessibility perspective: every page needs only one h1 and the next level of headlines needs to be h2 and then h3 and so on. If that is too big, we use the class h5 or h6 here: `<h2 class="h5">`

File: frontend/src/app/recipe/components/detail/recipe-detail.component.scss [hunk 5]
```diff
@@ -1,4 +1,5 @@
 :host {
+    padding-top: 8px;
```
Comment (crookoo) [inline frontend/src/app/recipe/components/detail/recipe-detail.component.scss:2]: Maybe even a bit more padding. I suggest to use 1rem which is 16 pixels here.

**Review by crookoo — CHANGES_REQUESTED** [review]

**Review by crookoo — APPROVED** [review]

**Pull request merged**